contracts/Super1155.sol

```diff
@@ -25,7 +26,7 @@ import "./proxy/StubProxyRegistry.sol";
 
   July 19th, 2021.
 */
-contract Super1155 is PermitControl, ERC165Storage, IERC1155, IERC1155MetadataURI {
+contract Super1155 is Context, ERC165Storage, IERC1155, IERC1155MetadataURI, ISuper1155, PermitControl {
```

Comment:
Does `PermitControl` still inherit from `Ownable` and `Context`? If so, why do we need explicit `Context` inheritance here?